_posts/2022-06-03-gaps-in-bitcoin-dev-funding.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "The gaps in bitcoin open-source funding"
+date:   2022-06-03 09:57:39 -0500
+categories: bitcoin funding grants
+---
+
+A few people reached out about my post on my [Guide for Grant Seekers](/bitcoin/funding/grants/grants-bitcoin-open-source/) with some critical feedback about promoting work for free culture. To clarify, this guide was not meant to be an ideal vision for the future, it was meant to reflect the present and help grant seekers achieve their goals.
+
+At the end of the post, I mentioned that this doesn't serve a lot of different groups. In fact, I think it serves a limited population which is why we see such geographic density of bitcoin FOSS devs. People that cannot do work for free over a long period aren't going to be able to make significant progress on becoming a FOSS dev quickly. Working on an all-consuming start-up, raising a family, or living in a place with limited electricity/internet will put grant seekers at a significant disadvantage. Because the proof of work is lower for other projects, this also means that, by default, they are more inclusive. I am biased because free work is such a useful screen. It's almost unforgeable proof of intrinsic motivation.
```

Comment:
I think this sounds better
```suggestion
At the end of the post, I mentioned that this doesn't serve a lot of different groups. In fact, I think it serves a limited population which is why we see such geographic density of bitcoin FOSS devs. Those who cannot do work for free over a long period aren't going to be able to make significant progress on becoming a FOSS dev quickly. Working on an all-consuming start-up, raising a family, or living in a place with limited electricity/internet will put grant seekers at a significant disadvantage. Because the proof of work is lower for other projects, this also means that, by default, they are more inclusive. I am biased because free work is such a useful screen. It's almost unforgeable proof of intrinsic motivation.
```